spec/descriptions/tagSyntheticResults.md

```diff
@@ -129,14 +154,24 @@ It will be replaced by **tagFilterExpression**.
 **tagFilterExpression** It serves as a filter to narrow down return results. Its type can be either EXPRESSION or TAG_FILTER with 
 logical operators AND or OR.
 
-A payload only needs either tagFilters or tagFilterExpression as a filter, not both. 
+A payload only needs either tagFilters or tagFilterExpression as a filter, not both.
+
+Either tagFilters or tagFilterExpression can specify a custom property by its key and value.
+```
+"tagFilters":[{
+  "name":"synthetic.tags",
+  "key":"location",
+  "value":"Denver",
+  "operator":"EQUALS"
+}]
+```
 
 ### Sample payload to get a list of Synthetic test results with tagFilters
 ```json
 {
-  "syntheticMetrics":["response_time","response_size"],
+  "syntheticMetrics":["synthetic.metricsResponseTime","synthetic.metricsResponseSize"],
```

Comment:
For syntheticMetrics, both response_time and synthetic.metricsResponseTime work.